### ISSUE TITLE
New multihoster 

### ISSUE THREAD

**Issue by jedlu91** [pyload/pyload#3596]
Hi.
Who can add new multihoster site to pyload?
And when i create a ticket?
Thanks

**Comment by GammaC0de** [comment]
> Who can add new multihoster site to pyload?

Me  :innocent:

> And where can i create a ticket?

You just did

Please provide URL of the multihoster and a test account.
Send account user and password to nitzo2001 at yahoo dot com.

**Comment by jedlu91** [comment]
Thank you.
I sent email with information.
Thx

**Comment by GammaC0de** [comment]
How exactly do you download with it? where is the documentation?

**Comment by jedlu91** [comment]
Hi. I write you email from Ardarr

**Comment by jedlu91** [comment]
Hi. Did you see my email??

**Comment by GammaC0de** [comment]
Yes,
This multihoster is not simple, all the data is tranfered via websocket.

**Comment by jedlu91** [comment]
Hmm please ask me if you have any question.
Meybe i can hep you or my programmer.

**Comment by jedlu91** [comment]
Yes. We fix last problems and start with this. So why i wanna that my site will be visible in pyload etc

**Comment by GammaC0de** [comment]
OK, there are issues to be solved so pyload can use it:

First of all, 
All API calls should be synchronous, not asynchronous via websocket, i.e.:
Calling `/api/file-download/url-check` should return the filename, size and status as a JSON string, not via `fileUpdate` event through the websocket

Also I need APIs that returns the current list of suppoted hosters and API that returns when the user expires and how much traffic he have (traffic left)

**Comment by jedlu91** [comment]
What if programmer told me that we dont have API and now the wey is having to parse a website ?

**Comment by GammaC0de** [comment]
> What if programmer told me that we dont have API and now the wey is having to parse a website ?

All the information is returned via a websoket, so the is nothing to parse..
It's an AngularJS web app...
